recipes/meepmeep/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
This file is in the tarball. You don't need to add it here again. Conda-build will copy it over for you.

Reply:
Thanks, this is fixed now.